CLAUDE.md

```
@@ -48,10 +48,8 @@ xcodebuild -project MediaCleaner.xcodeproj -scheme MediaCleaner -destination 'pl
 
 ## Active Technologies
 - Swift 5.9+, iOS 17+, SwiftUI
-- SwiftData (existing schema + SwipeRecord model)
-- Photos framework, AVFoundation
-- Swift 5.9+ + SwiftUI, SwiftData, Photos framework, AVFoundation (001-ux-refactoring)
-- SwiftData (existing models: SwipeRecord, TrashItem, CompressionHistoryItem, ReviewSession, StorageSavings, CompressionSettings) (001-ux-refactoring)
+- SwiftData (models: SwipeRecord, TrashItem, CompressionHistoryItem, ReviewSession, StorageSavings, CompressionSettings, Cache models)
+- Photos framework, AVFoundation, CryptoKit
 
 ## Recent Changes
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)